RELEASE.md

```diff
@@ -1,6 +1,15 @@
+# Release 0.3.1
+- Fix compatibility with Julia 0.6 [#341, #330, #293]
+- Support of Stan interface [#343, #326]
+- Fix Binomial distribution for gradients. [#311]
+- Stochastic gradient Hamiltonian Monte Carlo [#201]
```

Comment:
There is also: Stochastic Gradient Langevin Dynamics [https://github.com/yebai/Turing.jl/pull/327]

Reply:
and PIMH & PMMH: https://github.com/yebai/Turing.jl/pull/364

Reply:
Right - I will also add these items to RELEASE.md.

Reply:
Nice